docs/typescript.md

```diff
@@ -133,3 +133,48 @@ const argv = yargs.option('difficulty', {
 ```
 
 `argv.difficulty` will get  type `'normal' | 'nightmare' | 'hell'`.
+
+# Advanced example
+
+```typescript
+import yargs from 'yargs';
+import fs from 'fs';
+
+const args = yargs(process.argv.slice(2))
+    .scriptName('example-cli-program')
+    .epilog('Copyright (C) 2022 CREATOR')
+    .usage('$0 <file> [options]')
+    .showHelpOnFail(true)
+    .example([
+      ['$ node $0/dist/cli.js file.json -fv true -d output/'],
+      ['$ node $0/dist/cli.js file.xml -v true -f false -d output/']
+      ['$ node $0/dist/cli.js -d output/ file.xml']
+    ])
+    .env('EXAMPLE_CLI')
+    .positional('file', {type: 'string', describe: 'File to be handled'})
```

Comment:
Should we add a command other than the default command `$0`, to demonstrate how to configure positionals for a sub command?

I think this is a common use case.